app/src/main/res/values/strings.xml

```diff
@@ -128,8 +128,8 @@
         This is an beta feature where one can get manga recommendations.
         This is a recommendation system outside of MangaDex, and works by matching by genres,
         demographics, and then using term frequency–inverse document frequency (Tfidf) to get the
-        similarity of two manga\'s descriptions. You will need to download from the below
-        github the recommendation JSON manually.
+        similarity of two manga\'s descriptions. This file will be auto downloaded every 24 hours.
```

Comment:
... descriptions.  When enabled this file will download immediately and then auto download a refreshed file every 24 hours.